test/metabase/cmd/dump_to_h2_test.clj

```diff
@@ -29,3 +30,8 @@
                   :let     [file (io/file filename)]]
             (when (.exists file)
               (io/delete-file file))))))))
+
+(deftest cmd-dump-to-h2-returns-code-from-dump-test
```

Comment:
I don't know if testing this makes sense.  Covecod complained the cmd.dump-to-h2 command was not being tested properly so I added this, but I don't have a clear idea if this makes sense at all. Questions:

- I don't see how `dump-to-h2/dump-to-h2!` might return a value different than nil. I always see last expressions being prints. Maybe there's some pathway though.
- The command itself is in cmd.clj. If we leave the test, should I move it to `test/metabase/cmd.clj` ? It's weird that we don't have any yet, so that's why I ask.